### PR TITLE
Mm sharpening viewers

### PR DESCRIPTION
**Pull request by mmmtnez**
Hi all,

in addition to previous changes, here your are some changes in protocols involved in map sharpening:

    1)  xmipp3/protocols/protocol_postProcessing_deepPostProcessing.py: Inclusion of origin of coordinates.
    2)  xmipp3/protocols/protocol_volume_local_sharpening.py: Typo corrections.
    3)  xmipp3/viewers/__init__.py: Inclusion of two new classes of ChimeraX viewers.
    4)  xmipp3/viewers/viewer_deepEMHancer.py: New ChimeraX viewer for protocol xmipp3/protocols/protocol_postProcessing_deepPostProcessing.py (origin checked).
    5)  xmipp3/viewers/viewer_local_sharpening.py: New ChimeraX viewer for protocol xmipp3/protocols/protocol_volume_local_sharpening.py (origin checked).

Tests:

scipion3 tests xmipp3.tests.test_protocols_localdeblur.TestLocalDeblur
scipion3 tests xmipp3.tests.test_protocols_localdeblur.TestDeepVolPostProcessing

I guess the above test is:
    scipion3 tests xmipp3.tests.test_protocols_deepVolPostprocessing.TestDeepVolPostProcessing

### CODE REVIEW

File: xmipp3/viewers/viewer_deepEMHancer.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+from xmipp3.protocols.protocol_postProcessing_deepPostProcessing import \
+    (XmippProtDeepVolPostProc)
+from pwem.viewers.viewer_chimera import (Chimera, sessionFile)
+from pyworkflow.viewer import DESKTOP_TKINTER, Viewer
+
+class viewerXmippProtDeepVolPostProc(Viewer):
+    """ Visualize the input and output volumes of protocol XmippProtDeepVolPostProc
+        with ChimeraX (3D).
+        The axes of coordinates x, y, z will be shown"""
+    _label = 'viewer deepEMHancer'
```
Comment (rsanchezgarc) [inline xmipp3/viewers/viewer_deepEMHancer.py:44]: Tiny misspelling.

deepEMHancer -> deepEMhancer

**Review by rsanchezgarc — APPROVED** [review]

File: xmipp3/viewers/viewer_deepEMHancer.py (hunk 1 above)
Comment (mmmtnez) [reply xmipp3/viewers/viewer_deepEMHancer.py]: Done!

**Review by dmaluenda — APPROVED** [review]
The only concern I have here is about the new viewer files for deepEMhancer and localSharpening. If the are tested I agree on merge this PR

**Comment by DStrelak** [comment]
I have to agree with @dmaluenda . This release keeps snowballing new and new things.

**Comment by mmmtnez** [comment]
> I have to agree with @dmaluenda . This release keeps snowballing new and new things.

Sorry David, but I need specific viewers for those two protocols since I am updating the model building tutorial and I would like to include them.

**Comment by rmarabini** [comment]
I just check these PR. It is pretty straight forward. Mainly 2 new viewers. Since both viewer are similar a possible solution would be to create a generic chimera viewer that accepts a list of volumes.  Nevertheless, I think they are OK as they are.

**Pull request merged**